### DISCUSSION TITLE
No bootable SD card detected and No .x1p files found on SD card

### DISCUSSION THREAD

**Discussion by meckarmulle** [X1Plus/X1Plus#190] [Q&A]
Hi
When my X1 whith x1 plus booted up todag I got this message " No bootable SD card detekted" Then J did go to the Startup option and tryed "Start X1plus installer and gott the message "No .x1p files found on SD card"

That is probably my fault, I took out the card by mistake on a running X1.

What is the best way from here? Do an “Emergenscy recovery console? Or do I need to do an complete reinstall?

**Comment by LJKraus** [comment]
The card is apparently garbled, it might be possible to reformat it but the safe bet would be to reinstall X1P on a fresh FAT32 microSD.

**Comment by meckarmulle** [comment]
Thanks, you are probely right, I will do that

**Comment by yebo29** [comment]
I just ran across this issue. Initial install went fine. Powered down the printer and a few days later (today) it said that no bootable SD card found and it has me going through the install steps again. It's a brand-new SandDisk High Endurance card and I formatted it using the X1C screen before installing X1 Plus. I usually just cut power to the printer. Do I need to shut it down first via the UI? Not sure if there is such an option.

Thoughts?

**Comment by yebo29** [comment]
I had to completely uninstall X1 Plus and abandon it. Back to latest stock firmware. The last straw was when it happened mid-print. Print stopped and I was presented with that error and the screen to try and reinstall it. That was the 4th time it happened. Don't know what else to do.

**Reply by psiberfunk** [reply]
It really seems like something is either bad with your card or possibly your card reader that's making it unreliable... which is very unfortunate.  X1Plus has been rock stable for most people, and you should be able to just cut power to the printer, it doesn't really act any differently than stock in that regard.   I've been using X1+ for nearly a year or so and never had this issue :/

**Reply by irixirix** [reply]
I have had the same issue two SanDisk 512GB ultras I have tested with my camera and other devices work fine. Formatted in the x1c, tried putting the x1p and other files back on there- nada. Bootloader refused to see the card. Can the pre boot environment not see a 512GB fat32 extended volume? This is way bigger than standard fat32 would support. I am going to dig out a smaller SD card later for testing.

**Reply by irixirix** [reply]
Yep that was 100% the issue. 512GB cards formatted with rufus were a no go- however 256GB and smaller worked fine. Might want to update the readme on this fellas. Love the project.

**Reply by yebo29** [reply]
Okay, that's a good data point! I did think that perhaps the card size may have something to do with it. I'll try with a smaller card and report back. Thanks!
